src/sidebar/services/local-storage.ts

```diff
@@ -54,51 +51,38 @@ export class LocalStorageService {
 
   /**
    * Look up a value in local storage.
-   *
-   * @param {string} key
-   * @return {string|null}
    */
-  getItem(key) {
+  getItem(key: string): string | null {
     return this._storage.getItem(key);
   }
 
   /**
    * Look up and deserialize a value from storage.
-   *
-   * @param {string} key
    */
-  getObject(key) {
+  getObject<T = any>(key: string): T | null {
```

Comment:
If it isn't too much trouble to refactor, you could also use `T = unknown` here and force callers to specify what they expect or convert it.

Reply:
There was too many things "breaking", haha, so I kept it simple. I'll check again if I can do that with not a lot of impact

Reply:
OK. If it looks like it will be a significant extra change, feel free to leave this for another time.

Reply:
I have changed it. The linter was a bit alarming, but it was actually being called only in two places.